modules/layers/src/text-layer/multi-icon-layer/multi-icon-layer.js

```diff
@@ -29,8 +29,10 @@ const DEFAULT_GAMMA = 0.2;
 const DEFAULT_BUFFER = 192.0 / 256;
 
 const defaultProps = {
+  getLineLength: {type: 'accessor', value: x => x.lineLength || 0},
```

Comment:
- Add comments to explain what these accessors are
- Make the default a constant for better perf for those who are using this layer standalone